Functions/xASL_init_FileSystem.m

```diff
@@ -159,6 +159,8 @@
     end
     x.P.Pop_Path_MaskVascular = fullfile(x.D.PopDir, ['MaskVascular_' x.P.SubjectID '_' x.P.SessionID '.nii']);
     x.P.Path_Pop_MaskSusceptibility = fullfile(x.D.PopDir, ['rMaskSusceptibility_' x.P.SubjectID '_' x.P.SessionID '.nii']);    
+	
+	x.P.Path_ASL4D_aslcontext = fullfile(Path{2},'ASL4D_aslcontext.tsv');
```

Comment:
This filename is incorrect, in ExploreASL we currently exchange `asl` for `ASL4D`. So to be consistent:

BIDS -> legacy
`_asl.json` -> `ASL4D.json`
`_aslcontext.tsv` -> `ASL4Dcontext.tsv`

imho

Reply:
Very good comment! But we need to fix this everywhere ;) See this 26fa3e2d
Note that to properly test this, we will need to change flavors as well, and we can only do the full test upon merging with `develop` that has some new edits in import.

Reply:
Sure, we will revamp this anyway when going fully to BIDS. But nice if we can keep ourselves adherent to BIDS as much as possible, that will make the future changes smaller.

Reply:
Sure. But can you resolve this?! ;)